app/src/main/java/com/lucas/medicaltools/MainActivity.kt

```diff
@@ -100,12 +89,17 @@ class MainActivity() : MviActivity<MainView, MainPresenter>(), MainView {
     }
 
     private fun renderRecyclerView(medicalTools: List<MedicalTool>?) {
```

Comment:
- RenderStateX for data.
- Do not show spinner here, you have the data, hide the spinner.